configs/hrnet/fcos_hrnetv2p_w32_gn-head_4x4_1x_coco.py

```diff
@@ -1,4 +1,6 @@
 _base_ = '../fcos/fcos_r50_caffe_fpn_gn-head_4x4_1x_coco.py'
+img_norm_cfg = dict(
```

Comment:
This will not change the real img_norm_cfg used during training and testing because img_norm_cfg here is an intermediate variable, if you modify img_norm_cfg, you should also modify `train_pipeline` and `test_pipeline`